### PR TITLE
pre-first release review - DO NOT MERGE

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/multiplesequencealign/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/multiplesequencealign _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,83 +21,67 @@
...
 
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!
-
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources.The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/proteinfold/results).
```
Comment (maxulysse) [inline README.md:26]: you're missing this bit from the TEMPLATE

File: README.md [hunk 2]
```diff
@@ -21,83 +21,67 @@
...
 ![Alt text](docs/images/nf-core-msa_metro_map.png?raw=true "nf-core-msa metro map")
 
-1. **Collect Input Information**: computation of summary statistics on the input fasta file, such as the average sequence similarity across the input sequences, their length, etc. Skip by --skip_stats.
-2. **Guide Tree**: (Optional, depends on alignment tools requirement) Renders a guide tree.
-3. **Align**: Runs one or multiple MSA tools in parallel.
-4. **Evaluate**: The obtained alignments are evaluated with different metrics: Sum Of Pairs (SoP), Total Column score (TC), iRMSD, Total Consistency Score (TCS), etc. Skip by --skip_eval.
-5. **Compress**: As the final MSA files are very large, compression tools will be used before storing the final result. Skip by --skip_compress.
-
-Available GUIDE TREE methods:
+In a nutshell, the pipeline performs the following steps:
```
Comment (maxulysse) [inline README.md:26]:
```suggestion
The pipeline performs the following steps:
```

File: docs/extending.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+# Extending nf-core/multiplesequencealign
```
Comment (maxulysse) [inline docs/extending.md:1]: Isn't this covered by the Contributing Guidelines in `.github/CONTRIBUTING.md`?

File: docs/images/nf-core-msa_metro_map.png
Comment (maxulysse) [inline docs/images/nf-core-msa_metro_map.png:1]: The logo seems a bit off

File: docs/extending.md (hunk 3 above)
Comment (luisas) [reply docs/extending.md:1]: Not completely - here it specifies precisely how to do it for this specific pipeline (which parts of the workflow to touch etc). I added now a pointer to CONTRIBUTING from the extending

File: README.md (hunk 1 above)
Comment (luisas) [reply README.md:26]: uh ups! added :)

File: docs/images/nf-core-msa_metro_map.png
Comment (luisas) [reply docs/images/nf-core-msa_metro_map.png:1]: upsi, updated

**Comment by luisas** [comment]
dev branch was merged with #158 . 

I will close this one :)

**Pull request closed without merging**